.ci/static-checks.sh

```diff
@@ -530,7 +530,13 @@ check_url()
 
 	local ret
 
-	{ curl -sIL -H "Accept-Encoding: zstd, br, gzip, deflate" --max-time "$url_check_timeout_secs" \
+	# Authenticate for github to increase threshold for rate limiting
+	local github_arg=()
+	if [[ "$url" =~ github\.com && -n $GITHUB_USER && -n $GITHUB_TOKEN ]]; then
```

Comment:
Please double-quote `GITHUB_USER` and `GITHUB_TOKEN` as `-n "$GITHUB_USER" && -n "$GITHUB_TOKEN"`.